CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/panes/PluginReporterPane.java

```diff
@@ -90,6 +91,8 @@
     private static final int MAXIMUM_REPORT_PANES = 300;
 
     public PluginReporterPane() {
+        
+        this.setStyle((String.format("-fx-font-size:%d;", FontUtilities.getOutputFontSize())));
```

Comment:
You shouldn't need to do this here - it should be handled by `JavaFxTopComponent` and `ListeningTopComponent`.